packages/react-components/react-virtualizer/src/components/VirtualizerScrollView/VirtualizerScrollView.ts

```diff
@@ -0,0 +1,35 @@
+import { VirtualizerScrollViewProps } from './VirtualizerScrollView.types';
+import { useVirtualizerScrollView_unstable } from './useVirtualizerScrollView';
+import { renderVirtualizerScrollView_unstable } from './renderVirtualizerScrollView';
+import { useVirtualizerScrollViewStyles_unstable } from './useVirtualizerScrollViewStyles';
+import { useStaticVirtualizerMeasure } from '../../Hooks';
+import * as React from 'react';
+
+/**
+ * Virtualizer ScrollView
+ */
+
+export const VirtualizerScrollView: React.FC<VirtualizerScrollViewProps> = (props: VirtualizerScrollViewProps) => {
+  const scrollRef = React.useRef<HTMLElement | null>(null);
+  const { virtualizerLength, bufferItems, bufferSize } = useStaticVirtualizerMeasure(
```

Comment:
you should call this hook in `useVirtualizerScrollView` additional hooks in the main component breaks the recomposition pattern because users would need to be aware of extra hooks that are not the state, styles and renderer